src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -361,7 +361,8 @@ public class GenerateProtoTask extends DefaultTask {
     Preconditions.checkState(state == State.FINALIZED, 'doneConfig() has not been called')
 
     ToolsLocator tools = project.protobuf.tools
-    Set<File> protoFiles = inputs.sourceFiles.files
+    // Sort to make sure files are in a consistent order
+    List<File> protoFiles = inputs.sourceFiles.files.sort()
```

Comment:
Still trying to understand what the problem is. This will only affect the order of the proto files passed to the `protoc` command line. What output is changed by that?